### PR TITLE
cassandra-5.0beta1 image

### PR DESCRIPTION
**Pull request by conorbros**

### CODE REVIEW

**Comment by rukai** [comment]
You'll need to add publish logic to `.github/workflows/release.yaml` for cassandra 5

**Comment by rukai** [comment]
We should name this 5.0-beta1 and depend on 5.0-beta1 upstream image, instead of just `5.0`.
Otherwise changes upstream may broke our tests.

**Comment by rukai** [comment]
I have tested this works with the sample docker-compose's in the readme, so looking good so far.

File: .github/workflows/release.yaml [hunk 1]
```diff
@@ -28,3 +28,8 @@ jobs:
           IMAGE_NAME=shotover/cassandra-test:3.11.13-${GITHUB_REF/refs\/tags\//}
           docker build -t $IMAGE_NAME .
           docker push $IMAGE_NAME
+
+          cd ../cassandra-3.11.13
```
Comment (rukai) [inline .github/workflows/release.yaml]: wrong path

**Review by rukai — APPROVED** [review]
Looks good!

**Review by cjrolo — APPROVED** [review]

**Pull request merged**